RevenueCatUI/CustomerCenter/Data/CustomerCenterTestData.swift

```diff
@@ -0,0 +1,91 @@
+                id: "ownmsldfow",
+                title: CustomerCenterData.LocalizedString(en_US: "Didn't receive purchase"),
+                type: .missingPurchase,
+                promotionalOffer: nil,
+                feedbackSurvey: nil
+            ),
+            CustomerCenterData.HelpPath(
+                id: "nwodkdnfaoeb",
+                title: CustomerCenterData.LocalizedString(en_US: "Request a refund"),
+                type: .refundRequest,
+                promotionalOffer: CustomerCenterData.PromotionalOffer(
+                    iosOfferId: "rc-refund-offer",
+                    eligibility: CustomerCenterData.Eligibility(firstSeen: "> 30")
+                ),
+                feedbackSurvey: nil
+            ),
+            CustomerCenterData.HelpPath(
+                id: "nfoaiodifj9",
+                title: CustomerCenterData.LocalizedString(en_US: "Change plans"),
+                type: .changePlans,
+                promotionalOffer: nil,
+                feedbackSurvey: nil
+            ),
+            CustomerCenterData.HelpPath(
+                id: "jnkasldfhas",
+                title: CustomerCenterData.LocalizedString(en_US: "Cancel subscription"),
+                type: .cancel,
+                promotionalOffer: nil,
+                feedbackSurvey: CustomerCenterData.FeedbackSurvey(
+                    title: CustomerCenterData.LocalizedString(en_US: "Why are you cancelling?"),
+                    options: [
+                        CustomerCenterData.FeedbackSurvey.Option(
+                            id: "iewrthals",
+                            title: CustomerCenterData.LocalizedString(en_US: "Too expensive"),
+                            promotionalOffer: CustomerCenterData.PromotionalOffer(
+                                iosOfferId: "rc-cancel-offer",
+                                eligibility: CustomerCenterData.Eligibility(firstSeen: "> 14")
+                            )
+                        ),
+                        CustomerCenterData.FeedbackSurvey.Option(
+                            id: "qklpadsfj",
+                            title: CustomerCenterData.LocalizedString(en_US: "Don't use the app"),
+                            promotionalOffer: CustomerCenterData.PromotionalOffer(
+                                iosOfferId: "rc-cancel-offer",
+                                eligibility: CustomerCenterData.Eligibility(firstSeen: "> 7")
+                            )
+                        ),
+                        CustomerCenterData.FeedbackSurvey.Option(
+                            id: "jargnapocps",
+                            title: CustomerCenterData.LocalizedString(en_US: "Bought by mistake"),
+                            promotionalOffer: nil
+                        )
+                    ]
+                )
+            )
+        ],
+        title: CustomerCenterData.LocalizedString(en_US: "How can we help?"),
+        supportEmail: "ryan@revenuecat.com",
```

Comment:
😄

Reply:
lol will update this to prevent spam lol